# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1367555275 14400
#      Fri May 03 00:27:55 2013 -0400
# Node ID 2c11d7c6cc211630980d2de735d83a118d59bd6c
# Parent  189241a7c3a952d4ff0bdbcc879558d44cbf3827
turn off gdb thread messages by default

* gdbinit: Disable thread begin/end messages.

diff --git a/etc/gdbinit b/etc/gdbinit
--- a/etc/gdbinit
+++ b/etc/gdbinit
@@ -1,8 +1,13 @@
+## Disable the next line if you really need to see the
+## thread create/exit messages.
+
+set print thread-events off
+
 ## Helpful macros for debugging Octave.
 
 ############################################################
 ## Display a dim-vector object.
 
 define display-dims
   echo ndims: \ 
   output $arg0.rep[-1]
